osu.Game/Database/BeatmapExporter.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Platform;
+using osu.Game.Beatmaps;
+
+namespace osu.Game.Database
+{
+    public class BeatmapExporter : LegacyArchiveExporter<BeatmapSetInfo>
+    {
+        public BeatmapExporter(Storage storage)
+            : base(storage)
+        {
+        }
+
+        protected override string FileExtension => @".osz2";
```

Comment:
I just remembered that `.osz2` is being used by osu! stable for files in the 'SubmissionCache' so that's kind of a conflict. Feel free to suggest better extension names.